spring-cloud-azure-samples/spring-cloud-azure-data-jpa-sample/README.adoc

```diff
@@ -0,0 +1,72 @@
+= Spring Cloud Azure Sql Database Starter Sample
+
+This starter is based on `spring-boot-starter-jpa`. It fetches and auto configures data source properties via Azure Sql
```

Comment:
This sample application demonstrates how to use Spring Data JPA with Spring Cloud Azure SQL Server Starter.